IBM/carbon-addons-iot-react
# [Table] add padding removed by Carbon upgrade

Closes #1152 closes #1157

**Summary**

Table headers that are in a sortable table have had their padding removed. This PR restores padding.

**Change List (commits, features, bugs, etc)**

- added padding to _table-head.scss

**Acceptance Test (how to verify the PR)**

- Go to story at ?path=/story/watson-iot-table--stateful-example-with-expansion-and-column-resize
- open inspector and inspect Table header that does not have sorting
- see that padding is applied to all Table headers
